eng/Signing.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <!--
+    These are third party libraries that we use in Arcade. We need to sign them even if they
```

Comment:
Is this file necessary? Can it be removed?

Reply:
Removed